font-types/src/bbox.rs

```diff
@@ -14,3 +16,12 @@ pub struct BoundingBox<T> {
     /// region.
     pub y_max: T,
 }
+
+impl<T> BoundingBox<T> where T: MulAssign + Mul + Copy {
+    pub fn scale(&mut self, factor : T)  {
```

Comment:
Changing this to take `&self` and return a new copy of the scaled bounding box would make this easier to use when chaining calls, e.g. `map()`. Also lets us remove the `MulAssign` bound.

Reply:
Returning a new one, `MulAssign` bound removed.